### PR TITLE
fix: bump version of minimatch due to security issue PRISMA-2022-0039

### PR DESCRIPTION
**Pull request by opravil-jan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,11 +26,10 @@
   },
   "preferGlobal": true,
   "dependencies": {
-    "filelist": "^1.0.1",
```
Comment (kibertoad) [inline package.json]: why was this removed?

**Comment by kibertoad** [comment]
is it compatible with node 8?

**Comment by opravil-jan** [comment]
Why with node 8? In package.json is written you support 10 and higher versions? 

In node 8 does not pass all tests when you run in on main branch.

```
opravil@work-notebook:~/Projects/nodejs/x-delete/jake$ node -v
v8.17.0
opravil@work-notebook:~/Projects/nodejs/x-delete/jake$ npm run test

> jake@10.8.4 test /home/opravil/Projects/nodejs/x-delete/jake
> ./bin/cli.js test

Starting 'test:unit'...
Finished 'test:unit' after 4 ms
Starting 'test:integration'...


  namespace

    ✓ resolve namespace by relative name

    ✓ resolve task in sub-namespace by relative path
    ✓ prefer local to top-level
    ✓ does resolve top-level
    ✓ absolute lookup works from sub-namespaces
    ✓ resolution miss with throw error

  parseargs
    ✓ long preemptive opt and val with equal-sign, ignore further opts
    ✓ long preemptive opt and val without equal-sign, ignore further opts
    ✓ long preemptive opt and no val, ignore further opts
    ✓ preemptive opt with no val, should be true
    ✓ preemptive opt with no val, should be true and ignore further opts
    ✓ preemptive opt with val, should be val
    ✓ -f expects a value, -t does not (howdy is task-name)
    ✓ different order, -f expects a value, -t does not (howdy is task-name)
    ✓ -f expects a value, -t does not (foo=bar is env var)
    ✓ -f expects a value, -t does not (foo=bar is env-var, task-name follows)
    ✓ -t does not expect a value, -f does (howdy is task-name)
    ✓ --trace does not expect a value, -f does (howdy is task-name)
    ✓ --trace does not expect a value (equal), -f does (throw howdy away)


  19 passing (9ms)

  concurrent
    ✓  simple concurrent prerequisites 1 (2309ms)
    ✓  simple concurrent prerequisites 2 (2322ms)
    ✓  sequential concurrent prerequisites (2415ms)
    ✓  concurrent concurrent prerequisites (2281ms)
    ✓  concurrent prerequisites with subdependency (2093ms)
    ✓  failing in concurrent prerequisites (1867ms)

  fileTask
    ✓ where a file-task prereq does not change with --always-make (3489ms)
    ✓ concating two files (1907ms)
    ✓ where a file-task prereq does not change (3442ms)
    ✓ where a file-task prereq does change, then does not (4524ms)
    ✓ a preexisting file (3564ms)
    ✓ a preexisting file with --always-make flag (4029ms)
    ✓ nested directory-task (1915ms)

  fileUtils
    ✓ mkdirP
    ✓ rmRf
    ✓ rmRf with symlink subdir
    ✓ rmRf with symlinked dir
    ✓ cpR with same name and different directory
    ✓ cpR with same to and from will throw
    ✓ cpR rename via copy in directory
    ✓ cpR rename via copy in base
    ✓ cpR keeps file mode
    ✓ cpR keeps file mode when overwriting with preserveMode
    ✓ cpR does not keep file mode when overwriting
    ✓ cpR copies file mode recursively
    ✓ cpR keeps file mode recursively
    1) cpR copies directory mode recursively

  publishTask
    ✓ default task (1990ms)

  rule
    ✓ Rule.getSource
    ✓ rule w/o pattern (1754ms)
    ✓ rule w pattern w/o folder w/o namespace (1754ms)
    ✓ rule w pattern w folder w/o namespace (1745ms)
    - rule w pattern w folder w namespace
    - rule w chain w pattern w folder w namespace
    ✓ rule with source file not created yet (precedence) (1727ms)
    ✓ rule with source file now created (precedence) (1709ms)
    ✓ rule with source file modified (precedence) (2714ms)
    ✓ rule with existing objective file and no source  (should be normal file-task) (precedence) (1736ms)
    ✓ rule with source file not created yet (regexPattern) (1790ms)
    ✓ rule with source file now created (regexPattern) (1726ms)
    ✓ rule with source file modified (regexPattern) (2801ms)
    ✓ rule with existing objective file and no source  (should be normal file-task) (regexPattern) (1724ms)
    ✓ rule with source file not created yet (sourceFunction) (1763ms)
    ✓ rule with source file now created (sourceFunction) (1929ms)
    ✓ rule with source file modified (sourceFunction) (2723ms)
    ✓ rule with existing objective file and no source  (should be normal file-task) (sourceFunction) (1726ms)

  selfDep
    ✓ self dep const (1853ms)
    ✓ self dep dyn (1812ms)

  taskBase
    ✓ default task (3502ms)
    ✓ task with no action (1847ms)
    ✓ a task with no action and no prereqs (1809ms)
    ✓ a task that exists at the top-level, and not in the specified namespace, should error (1703ms)
    ✓ passing args to a task (1741ms)
    ✓ a task with environment vars (1771ms)
    ✓ passing args and using environment vars (1914ms)
    ✓ a simple prereq (1858ms)
    ✓ a duplicate prereq only runs once (1773ms)
    ✓ a prereq with command-line args (2601ms)
    ✓ a prereq with args via invoke (2023ms)
    ✓ a prereq with args via execute (1924ms)
    ✓ repeating the task via execute (1730ms)
    ✓ prereq execution-order (1758ms)
    ✓ basic async task (1937ms)
    ✓ promise async task (1883ms)
    ✓ failing promise async task (1724ms)
    ✓ that current-prereq index gets reset (1858ms)
    ✓ modifying a task by adding prereq during execution (1773ms)
    ✓ listening for task error-event (1701ms)
    ✓ listening for jake error-event (1850ms)
    ✓ listening for jake unhandledRejection-event (1755ms)
    ✓ large number of same prereqs (1946ms)
    ✓ large number of different prereqs (1866ms)
    ✓ large number of different prereqs (1702ms)
    ✓ modifying a namespace by adding a new task (1926ms)


  71 passing (2m)
  2 pending
  1 failing

  1) fileUtils
       cpR copies directory mode recursively:

      AssertionError [ERR_ASSERTION]: 448 == 493
      + expected - actual

      -448
      +493
      
      at Context.<anonymous> (file.js:220:12)



Finished 'test:integration' after 121486 ms
Starting 'test:integrationClobber'...
Finished 'test:integrationClobber' after 12 ms
Starting 'test'...
Finished 'test' after 0 ms
```

**Comment by kibertoad** [comment]
fair point. we need to drop outdated travis config and add GA for Node 10+

**Comment by opravil-jan** [comment]
What are the next steps to make this PR merged, please?
